viewer/packages/tools/src/Measurement/Measurement.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+export interface Measurement {
+  /**
+   * Add a measurement
+   * @param point Start point of the Measurement
+   */
+  add(point: THREE.Vector3): void;
+  /**
+   * Remove a measurement
+   */
+  remove(): void;
+  /**
+   * Update the measurement
+   * @param x screen X Position
+   * @param y screen Y Position
+   */
+  update(x?: number, y?: number, point?: THREE.Vector3): void;
```

Comment:
Update what? What happens if x and y is undefined? When point is undefined? Are X screen positions, or relative? This function is very confusing